packages/peregrine/lib/Price/__tests__/__snapshots__/price.spec.js.snap

```diff
@@ -48,7 +48,7 @@ Array [
     1
   </span>,
   <span>
-     
+     
```

Comment:
Not sure why this snapshot would have changed 🤔

Reply:
Yea and tests failed now so...maybe its a node version thing? 🤷